test/test_group.cpp

```diff
@@ -2623,10 +2623,6 @@ TEST(Test_Commit_Compression_Strings)
         auto set_s = obj.get_set<String>(col_key_set_string);
         auto dictionary_s = obj.get_dictionary(col_key_dict_string);
 
-        CHECK_EQUAL(list_s.size(), i + 1);
```

Comment:
This is not related to this change, but in one of the N runs I happened to hit a generation of the same string twice. Checking the size is not smart, It is OK to just fetch the value.